.github/workflows/e2e.yml

```diff
@@ -8,10 +8,16 @@ on:
   push:
     branches:
       - main
-      - release/**
+      - v0.3[478].x
+      - v[1-9][0-9]?.x
```

Comment:
You might need to enclose them in double-quotes ?

This should work `"v[0-9]+.[0-9]+.[0-9]+" `

See an example here:   https://github.com/cometbft/cometbft/blob/8bfd04a6f23eec1b41393f0d67228b3f5942511e/.github/workflows/testapp-docker.yml#L9C9-L9C36

Reply:
As far as I read, the quotes are only necessary if the string starts with a special character ('*', '!') that may break the yaml syntax